examples/with-tailwind/packages/ui/package.json

```diff
@@ -13,22 +13,23 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "tailwindcss -i ./src/styles.css -o ./dist/index.css",
-    "lint": "eslint src/",
-    "dev": "tailwindcss -i ./src/styles.css -o ./dist/index.css --watch",
-    "type-check": "tsc --noEmit"
+    "build": "npx @tailwindcss/cli -i ./src/styles.css -o ./dist/index.css",
```

Comment:
Nit, but this can just be `tailwindcss` (the `@tailwindcss/cli` package installs a bin with that name) to avoid the `npx`

https://github.com/tailwindlabs/tailwindcss/blob/next/packages/%40tailwindcss-cli/package.json#L19

Reply:
This is how they say to do it in their doc... 🤔
https://tailwindcss.com/docs/installation/tailwind-cli

Reply:
Yea, it's going to be slower that way but will _always_ work regardless of where you installed `@tailwindcss/cli`. But fine keeping it as is to match their docs.

Reply:
Huh, I had thought I tried it as `tailwindcss` before but maybe I had done just `tailwind`. Good looks.